### PR TITLE
Reorder menu to fit the new 3.3. layout

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+|3.3    |\_eh33     |master     |
+
+Of course, if for some reason you want a specific tag for a version, just create a worktree based on the tag instead.
+
+### Jekyll
+
+The website is rendered by [Jekyll](https://jekyllrb.com/). It is built locally and the result is pushed to the 
+[website repository](https://github.com/ehcache/ehcache.github.io).
+
+To generate and view (locally serve) the site `jekyll serve -w` then point your browser at `http://localhost:4000`. The `-w`
```
Comment (ljacomet) [inline README.md]: the -w is not needed, Jekyll does the dynamic generation even without it

File: _layouts/docs33_page.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    <!-- sidebar -->
+    <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar" role="navigation">
+        <ul class="nav">
+          <li class="submenu"><a href="index.html">Docs Overview</a></li>
+        </ul>
+
+        Basic topics:
+        <ul class="nav">
+          <li class="submenu"><a href="getting-started.html">Getting Started</a></li>
+          {% if page.ehc_version != '3.0' %}
```
Comment (ljacomet) [inline _layouts/docs33_page.html]: This `if` are obsolete

File: README.md (hunk 1 above)
Comment (henri-tremblay) [inline README.md]: Ok. Didn't know that

**Comment by henri-tremblay** [comment]
ehcache3#1912 should be merged first. Then the site can be deployed right away unless that are things I don't know

**Comment by ljacomet** [comment]
Ok, then the answer is no, this cannot be merged before we do the 3.3.1 release. The website does not deploy the docs out of master, but out of a release tag.

**Pull request merged**